docs/UserGuide.md

```diff
@@ -300,35 +355,34 @@ Examples:
 
 ### Adding a lesson: `add -l`
 
-Adds a new lesson to TutorAid.
+Adds a new lesson to TutorAid. The lesson's capacity, price and timing are optional details for tutors to include.
 
 Format: `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIMING]`
 
 * The lesson name should only contain alphanumeric characters and spaces.
 * If provided, the lesson's capacity must be a **positive integer** 1, 2, 3, …
-* If provided, the lesson's price must be a **non-negative number** with either 0 or 2 decimal places. Examples of a valid price are `80` and `85.50`.
+* If provided, the lesson's price must be a **non-negative number** with either 0 or 2 decimal places. Examples of a valid price are `80` and `85.50`. 
+* The lesson's capacity, price and timing are optional details for tutors to include.
 
 Examples:
 * `add -l n/P6 Maths c/20 p/80 t/Monday 1200-1400`
 
-> :bulb: The lesson's capacity, price and timing are optional details for tutors to include.
-
 ### Deleting a lesson : `del -l`
 
-Deletes the specified lesson with the given lesson index from TutorAid.
+Deletes the lesson at the specified index.
 
 Format: `del -l LESSON_INDEX`
 
 * Deletes the lesson at the specified `LESSON_INDEX`.
-* `LESSON_INDEX` refers to the index number shown in the displayed lesson list.
+* `LESSON_INDEX` refers to the index number shown in the lesson panel.
 * `LESSON_INDEX` must be a **positive integer** 1,2,3, …​
 
 Examples:
-* `del -l 3`
+* `del -l 3` deletes the 3rd lesson displayed in the lesson panel from TutorAid
```

Comment:
A very very minor nitpick here, but can consider adding a period `.` at the end at the sentence for consistency!